src/SFA.DAS.IdentifyDataLocks.Web/Infrastructure/DataLockService.cs

```diff
@@ -17,11 +17,13 @@ public class DataLockService
 
         public async Task<ApprenticeshipModel> GetActiveApprenticeship(long uln)
```

Comment:
I wondered if `GetActiveApprenticeship` is still the right name for this method now.  I came to the conclusion that it still makes sense, as there cannot be another active apprenticeship for the learner whilst one is paused.

Reply:
We could be more specific `GetActiveOrPausedApprenticeship` but I agree the existing name will work as well.